python/pyspark/errors/error-conditions.json

```diff
@@ -0,0 +1,1096 @@
+{
```

Comment:
The problem is that it has to be packaged together, and able to be uploaded into PyPI. Would be great if we can make sure that still works

Reply:
Yes, good call out. I was looking at `MANIFEST.in` and I believe this file should be included, but I will confirm.

Reply:
Confirmed. I've updated the PR description accordingly.